File: Algorithm/Algo1.cpp
#include <iostream>
using namespace std;

int linearSearch(int arr[], int key, int size)
{
    for (int i = 0; i < size; i++)
    {
        if (arr[i] == key)

            return i;
    }
    return -1;
}
int main()
{
    int arr[7] = {8, 10, 12, 14, 16, 18, 20};
    int key = 18;
    int ans = linearSearch(arr, key, 7);
    if (ans == -1)
    {
        cout << "Element not found";
    }
    else
    {
        cout << "Element found in index " << ans;
    }
}

File: Algorithm/Algo1.py
def linearSearch(arr,key):
    for i in range(len(arr)):
        if arr[i]==key:
            return i
    return -1

arr = [10,5,8,9,12]
key= 9
ans = linearSearch(arr,key)
if ans==-1:
    print("Element not found")
else:
    print("Element found in index",ans)